core/src/builder/transaction/creator.rs

```diff
@@ -662,7 +662,7 @@ pub async fn create_txhandlers(
         .get_txid()
         .to_byte_array();
 
-    let vout = kickoff_data.kickoff_idx + 1; // TODO: Extract directly from round tx - not safe
+    let vout = kickoff_data.kickoff_idx + 1;
     let watchtower_challenge_start_idx = (FIRST_FIVE_OUTPUTS + NUMBER_OF_ASSERT_TXS) as u16;
```

Comment:
```suggestion
    let vout = UtxoVout::Kickoff(kickoff_data.kickoff_idx as usize).get_vout();
    let watchtower_challenge_start_idx = UtxoVout::WatchtowerChallenge(0).get_vout() as u16;
```
I try to fix these when I see them